nominal/core/workbook.py

```diff
@@ -16,6 +17,9 @@ class Workbook(HasRid):
     title: str
     description: str
     run_rid: str | None
+    asset_rids: Sequence[str] | None
+    is_draft: bool
+    is_locked: bool
```

Comment:
since you can mutate these things with methods, can we turn these into API calls instead? basically don't want to allow for inconsistencies here